ProcessLib/HydroMechanics/CreateHydroMechanicsProcess.cpp

```diff
@@ -165,9 +165,9 @@ std::unique_ptr<Process> createHydroMechanicsProcess(
     auto media_map =
         MaterialPropertyLib::createMaterialSpatialDistributionMap(media, mesh);
 
-    std::vector<MaterialPropertyLib::PropertyType> requiredGasParams{
+    std::vector<MaterialPropertyLib::PropertyType> const requiredGasParams{
```

Comment:
sorry to open discussion agai. In [#2772](https://github.com/ufz/ogs/pull/2772/files#diff-04eb6b0a4a7982ff1ca36675aa6f79abR293-R296) we have decided to use `std::array` for [several](https://github.com/ufz/ogs/pull/2772#pullrequestreview-346439926) reasons.

Reply:
ok, I see the reasons. Going to fix it

Reply:
Thanks!